docs/understanding-rendering.md

```diff
@@ -28,6 +28,7 @@ getElements() {
 				<Sidebar  />
 			</RootElement>
 		</RootContainer>
+		<TheFold />
```

Comment:
What happens if some fool tries to stick `<TheFold />` in a `RootContainer`, or use it in a plain React component?